### ISSUE TITLE
SQL Error when select in DLF:Kollection a single Collection

### ISSUE THREAD

**Issue by BFallert** [kitodo/kitodo-presentation#381]
### Section "Nur diese Kollektionen anzeigen"

When I click on a collection in the  field from "**Available Items**" (so that the collection is moved into "**Selected Items**") i get an SQL error message, if i want to display the page in which the DLF:Kollektion element is located.

```
Oops, an error occurred!

An exception occurred while executing 'SELECT `tx_dlf_collections`.`index_name` 
AS `index_name`, `tx_dlf_collections`.`index_search` AS `index_query`, 
`tx_dlf_collections`.`uid` AS `uid`, `tx_dlf_collections`.`sys_language_uid` 
AS `sys_language_uid`, `tx_dlf_collections`.`label` AS `label`, 
`tx_dlf_collections`.`thumbnail` AS `thumbnail`, `tx_dlf_collections`.`description` 
AS `description`, `tx_dlf_collections`.`priority` AS `priority` 
FROM `tx_dlf_collections` WHERE (`tx_dlf_collections`.`uid` IN (1)) AND 
(`tx_dlf_collections`.`fe_cruser_id` = 0) AND (`tx_dlf_collections`.`pid` = 6) AND 
(((`tx_dlf_collections`.`sys_language_uid` IN (-1, 0)) OR 
(`tx_dlf_collections`.`sys_language_uid` = 0)) AND (`tx_dlf_collections`.`l18n_parent` = 0)) 
AND ((`tx_dlf_collections`.`deleted` = 0) AND (`tx_dlf_collections`.`hidden` = 0) 
AND (((`tx_dlf_collections`.`fe_group` = '') OR (`tx_dlf_collections`.`fe_group` IS NULL) 
OR (`tx_dlf_collections`.`fe_group` = '0') 
OR (FIND_IN_SET('0', `tx_dlf_collections`.`fe_group`)) OR (FIND_IN_SET('-1', 
`tx_dlf_collections`.`fe_group`))))) AND ((`tx_dlf_collections`.`deleted` = 0) AND 
(`tx_dlf_collections`.`hidden` = 0)) 
ORDER BY `FIELD(tx_dlf_collections`.`uid,1)` ASC': Unknown 
column 'FIELD(tx_dlf_collections.uid,1)' in 'order clause' 
```

If I remove the collection so that "**Selected Items**" is empty, the page will be displayed normally.

**Comment by sebastian-meyer** [comment]
This was is a known issue which was just fixed with #380

**Comment by sebastian-meyer** [comment]
In the upcoming release we are using Solarium to perform Solr searching. Solarium doesn't support using the MySQL `FIELD()` function in an `ORDER BY` statement.
